src/NavigationBar/index.js

```diff
@@ -0,0 +1,214 @@
+
+const NavigationBar = ({
+  backgroundColor,
+  shadow,
+  border,
+  borderColor,
+  borderWidth,
+  title,
+  menuItems,
+  firstMenuItem,
+  secondMenuItem,
+  thirdMenuItem,
+  fourthMenuItem,
+  fifthMenuItem,
+  profileImage,
+  _fonts,
+  editor,
+  _width,
+}) => {
+  const [activeMenuItem, setActiveMenuItem] = useState(
+    menuItems.defaultActiveMenuItem
+  )
+
+  const items = [
+    firstMenuItem,
+    secondMenuItem,
+    thirdMenuItem,
+    fourthMenuItem,
+    fifthMenuItem,
+  ]
+
+  const getBorderStyle = () => {
+    if (border) {
+      return {
+        borderBottomWidth: borderWidth,
+        borderBottomColor: borderColor,
+      }
+    }
+  }
+
+  const renderProfileImage = () => {
+    const { enabled, image, rounding, mobileText, actions } = profileImage
+    if (!enabled) {
+      return <View />
+    }
+
+    return (
+      <Image
+        source={image || titlePlaceholder}
+        style={{
+          width: 40,
+          height: 40,
+          borderRadius: rounding,
+          resizeMode: 'cover',
+          justifyContent: 'flex-end',
+        }}
+        onPress={actions}
+      />
+    )
+  }
+
+  const renderAdditionalItems = () => {
+    return <View />
+  }
+
+  const renderNavBar = () => {
+    let variant =
+      _width > DeviceBreakpoint.TABLET_BREAKPOINT ? 'desktop' : 'mobile'
+
+    let containerStyles = {
+      backgroundColor,
+      height: 76,
+      paddingTop: 32,
+      paddingRight: 20,
+      paddingLeft: variant === 'desktop' ? 20 : 12,
+      paddingBottom: 12,
+      flexDirection: 'row',
+      alignItems: 'center',
+      ...getBorderStyle(),
+      ...getShadowStyle(shadow),
+    }
+    if (!editor) {
+      containerStyles = {
+        ...containerStyles,
+        height: 106,
+        paddingTop: 50,
+        marginTop: -30,
+        ...getBorderStyle(),
+      }
+    }
+
+    // Logic for centering the title and menu items on mobile and desktop respectively
+    const centerStyles = {
+      position: 'absolute',
+      display: 'inline-flex',
+      justifyContent: 'center',
+      left: 0,
+      right: 0,
+      margin: '0 auto',
+    }
+
+    let menuItemsStyles = {
+      flexDirection: 'row',
+      marginLeft: menuItems.alignment === 'right' ? 'auto' : '',
+    }
+
+    if (menuItems.alignment === 'center') {
+      menuItemsStyles = {
+        ...menuItemsStyles,
+        ...centerStyles,
+      }
+    }
+
+    let mobileAlignment = title.universalLayout ? 'left' : title.mobileAlignment
+    let mobileTitleStyles = {
+      marginLeft: mobileAlignment === 'right' ? 'auto' : '',
+      paddingRight: '12px',
+      flexDirection: 'row',
+    }
+
+    if (mobileAlignment === 'center') {
+      mobileTitleStyles = {
+        ...mobileTitleStyles,
+        ...centerStyles,
+      }
+    }
+
+    if (variant === 'desktop') {
+      // render the title on the left, the menu items in the middle, and the profile image on the right
+      return (
```

Comment:
Getting these two jsx into separate components will make the logic/code here  more readable